apps/cli/src/commands/boot-challenger.ts

```diff
@@ -338,7 +338,7 @@ export function bootChallenger(cli: Vorpal) {
 
             }
 
-            clearInterval(assertionCheckInterval);
+            //clearInterval(assertionCheckInterval);
```

Comment:
Had to comment this out due to build error. I know there are other boot-challenger stories that have not been merged into tiny keys, so I did not create a story around this, but I can if it's needed.